# SOME DESCRIPTIVE TITLE.
# Copyright (C) 
# This file is distributed under the same license as the Algolib package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Algolib \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-08-16 05:55+0900\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../source/api/algolib.rst:2
msgid "algolib package"
msgstr ""

#: ../source/api/algolib.rst:5
msgid "Subpackages"
msgstr ""

#: ../source/api/algolib.rst:13
msgid "Submodules"
msgstr ""

#: ../source/api/algolib.rst:16
msgid "algolib.exceptions module"
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.AlgolibError:1
msgid "Bases: :py:class:`Exception`"
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.AlgolibError:1
msgid "Base exception for this library."
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.InvalidTypeError:1
msgid "Bases: :py:class:`~algolib.exceptions.AlgolibError`, :py:class:`TypeError`"
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.InvalidTypeError:1
msgid "Raised when an argument has an invalid type."
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.InvalidValueError:1
msgid "Bases: :py:class:`~algolib.exceptions.AlgolibError`, :py:class:`ValueError`"
msgstr ""

#: ../../src/algolib/exceptions.py:docstring of algolib.exceptions.InvalidValueError:1
msgid "Raised when an argument has an invalid value."
msgstr ""

#: ../source/api/algolib.rst:24
msgid "Module contents"
msgstr ""
